libfabric.vcxproj

```diff
@@ -474,4 +475,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>
```

Comment:
Yes, that's fine. I cloned your changes and the project can be opened in Visual Studio w/o problem (at least VS 2015 is fine with that)